docs/config.toml

```diff
@@ -34,9 +34,10 @@ home = [ "HTML", "RSS", "SearchIndex" ]
     { name = "Quickstart", url = "/spark-quickstart", weight = 100 },
     { name = "Docs", url = "/docs/latest", weight = 200 },
     { name = "Releases", pre = "relative", url = "../../releases", weight = 600 },
+    { name = "Roadmap", pre = "relative", url = "../../roadmap", weight = 997 },
     { name = "Blogs", pre = "relative", url = "../../blogs", weight = 998 },
     { name = "Talks", pre = "relative", url = "../../talks", weight = 999 },
-    { name = "Roadmap", pre = "relative", url = "../../roadmap", weight = 997 },
+    { name = "Vendors", pre = "relative", url = "../../vendors", weight = 1000 },
```

Comment:
Does this need to be a topnav item? The home page has a lot of marketing content and buttons already so what if we add a blurb about vendors somewhere there and have a button that says "Vendor List" which takes you to vendors.md?
<img width="1626" alt="Screen Shot 2022-07-29 at 7 05 12 AM" src="https://user-images.githubusercontent.com/43911210/181777445-0012ba8a-1ad6-4958-a5f3-7f5c19486169.png">

Reply:
You can see how those are created [here](https://github.com/apache/iceberg-docs/tree/main/landing-page/content/services).

Reply:
The home page body rather shows features of what Iceberg can do currently. Actually it begins with "What is Iceberg" and is kind of describing it further down. I don't really see how a vendor blurb could fit among these feature descriptions/examples.
IMHO a 'Vendors' link is better placed wherever external materials are already, and thus I think the place next to Blogs and Talks is more suitable. We could reposition Vendors elsewhere, but then I'd suggest to bring Blogs and Talks too with it. What do you think?

Reply:
You're making some great points ;)

I agree, let's keep it next to Blogs and Talks. 👍